app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -674,6 +685,18 @@ protected void setupWithStatus(StatusViewData.Concrete status,
         }
     }
 
+    private void setupCounters(StatusViewData.Concrete status) {
+        if (countFavorites!=null)
+            countFavorites.setText(status.getFavouritesCount()>0?String.format(Locale.getDefault(),"%d",status.getFavouritesCount()):"");
+
+        if (countReposts!=null)
+            countReposts.setText(status.getReblogsCount()>0?String.format(Locale.getDefault(),"%d",status.getReblogsCount()):"");
+
+        if (countReplies!=null)
+            countReplies.setText(status.getRepliesCount()>0?String.format(Locale.getDefault(),"%d",status.getRepliesCount()):"");
```

Comment:
The web UI does "0", "1", "1+" for replies - do we want to do the same?
What's the plan for dealing with many-digit counts on these?

Reply:
This is how it works in the Roma fork so carried over the functionality, but open to suggestions.